compute/etc/pgbouncer.ini

```diff
@@ -19,6 +19,8 @@ max_prepared_statements=0
 admin_users=postgres
 unix_socket_dir=/tmp/
 unix_socket_mode=0777
+; required for pgbouncer_exporter
+ignore_startup_parameters=extra_float_digits
```

Comment:
I wonder if we should document this? IIUC, this means that if someone sets https://postgresqlco.nf/doc/en/param/extra_float_digits/ as a connection string parameter, it won't work with the pooled connection string

It's unlikely that more than a few users will use it, but still. cc @danieltprice